src/common/dialog_commands.hpp

```diff
@@ -0,0 +1,127 @@
+// every phase in dialog can have some buttons
+// buttons are generalized on this level as commands
+// because non GUI/WUI client can also use them
+
+#pragma once
+
+#include <cstdint>
+#include <cstddef>
+#include <array>
+enum { COMMAND_BITS = 2,
+    MAX_COMMANDS = (1 << COMMAND_BITS) };
+
+//list of all button types
+enum class Command : uint8_t {
+    _NONE = 0, //none must be zero becouse of empty initialization of array
+    YES,
+    NO,
+    CONTINUE,
+    OK,
+    BACK,
+    RETRY,
+    PURGE_MORE,
+    REHEAT
```

Comment:
I would prefer to use ALL UPPERCASE only for preprocessor macros.

Reply:
done